src/main/java/ch/kalunight/zoe/command/create/RegisterCommand.java

```diff
@@ -38,32 +39,34 @@ public RegisterCommand() {
   }
   
   @Override
-  protected void executeCommand(CommandEvent event) {
+  protected void executeCommand(CommandEvent event) throws SQLException {

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'executeCommand(CommandEvent)' has an NPath complexity of 3456](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=4739924)